### PR TITLE
TGU: 2FA for admins, 2-person mech, hair gradients. Fish price halved!!

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

A guy in Reddit said that Fulpstation never updates so it catches dust therefore I'm doing a TGU to prove them wrong

Okaaay soooo

-Switchblades now switch (if we didn't already have this)
- FISH PRICE HALVED!!! ECONOMY RISING!! VERY GOOD!
- Admins can now edit expired/removed bans
- Hair dyes. Yes, aka hair gradients. Use a spray from library vendor or the colorist quirk.
- Shuttle insurance works
- 2FA for Admins. Yes, this edits database. @remove32 
- Emotes now have an audio cooldown
- Some logging stuff
- 2 person mech!!!!
- probably a lot more stuff idk if last TGU had the fix that made R&D Console not lag but if it didn't then here it is

### CODE REVIEW

**Comment by remove32** [comment]
the 2FA for admins thing - we'll need to look at how that is configured and set up because for TG i'm pretty sure it leverages their forum

**Comment by SgtHunk** [comment]
> the 2FA for admins thing - we'll need to look at how that is configured and set up because for TG i'm pretty sure it leverages their forum

Here's the PR for reference
https://github.com/tgstation/tgstation/pull/59467

**Comment by remove32** [comment]
applied the DB update
gonna have to investigate what i'd need to do for the actual auth part

File: code/datums/elements/turf_transparency.dm [hunk 1]
```diff
@@ -47,16 +48,20 @@
...
+
 	update_multiz(our_turf)
 
-/datum/element/turf_z_transparency/proc/on_multiz_turf_new(turf/our_turf, turf/T, dir)
+/datum/element/turf_z_transparency/proc/on_multiz_turf_new(turf/our_turf, turf/below_turf, dir)
 	SIGNAL_HANDLER
+
 	if(dir != DOWN)
 		return
+
```
Comment (remove32) [inline code/datums/elements/turf_transparency.dm:64]: mfw \r\n

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -81,7 +81,7 @@
 
 # Multiple Owners
 
-/icons/ @Twaticus @ShizCalev
+/icons/ @Twaticus @ShizCalev @Krysonism
```
Comment (JohnFulpWillard) [inline .github/CODEOWNERS:84]: wtf? this isnt needed, im already the owner of everything

**Comment by JohnFulpWillard** [comment]
This is porting wintercoat's GAGS, how is our prisoner wintercoat doing?

**Comment by SgtHunk** [comment]
We HAVE A PRISONER WINTERCOAT?

**Comment by JohnFulpWillard** [comment]
fulp_features/features/fulp_clothing/wintercoats/code/wintercoats.dm

**Comment by SgtHunk** [comment]
WHY DOES IT HAVE ARMOR
WHY DOES IT DEFINE VARIABLES ALREADY INHERITED
WHYY IS THE HOOD A /FULP SUBTYPE BUT THE COAT ISN'T
WHYYYYY

**Pull request merged**